src/common/options.cc

```diff
@@ -793,6 +793,12 @@ std::vector<Option> get_global_options() {
     .set_default("crypto_isal")
     .set_description("Crypto accelerator library to use"),
 
+    Option("openssl_engine_opts", Option::TYPE_STR, Option::LEVEL_ADVANCED)
+    .set_default("none")
```

Comment:
Is there any particular reason for going with `none` as default? Maybe leaving it empty would do?